grant/projects/subview.md

```diff
@@ -0,0 +1,99 @@
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Crab DVM currently does not have a specific DVM block explorer. For developers, will encounter obstacles when using the Crab network, especially when developers develop a Dapp. Subview can provide detailed block information and transaction information，developers can read and verify Solidity smart contracts and access pre-existing contracts to fast-track development.
+
+2. For users, Subview support for ERC20 and ERC721 tokens, you can easily view your asset statistics, transaction history, and gas fee usage, contract method calls and logs are decoded to provide additional information directly from the UI.
+
+### **How you will integrate this project into Crab:**
+
+Subview will deploy a Crab Network based explorer instance, which will scan past blocks and complete all block statistics. Once the scan is complete, crab users visit the subview to query the blockchain data they need.
+
+### **Team**
+
+- Shell ( Front-end Engineer )
+- Eason ( DB Engineer )
+
+### **Legal Structure**
+
+Currently Subview has no defined legal structure other then a Dapp created by Subview team.
+
+## **Team Code Repos**
+
+The version is not open source.
+
+## **Collaborative Grant Proposal and Current Development**
+
+### **Proposal**
+
+Subview provides a comprehensive, easy-to-use interface for users to view, confirm, and inspect transactions on EVM (Ethereum Virtual Machine).
+
+Subview will provide the following to Crab community:
+
+- crab.subview.xyz provides user access to crab network statistics
+- Allow other projects like Evolution Land, snowswap, subswap, metamask to use crab.subview.xyz as block explorer.
+
+## **Subview 1.0**
+
+The version of Subview is what is applying for CRAB grant program ( [Subview for Crab Network](https://crab.subview.xyz )
+), and the delivery scope has:
+
+- Block: Provide detailed information about each block.
+
+- Transaction: Provides analysis of transactions.
+
+- Smart contract interaction: Users can read and verify Solidity smart contracts and access pre-existing contracts to fast-track development. 
+
+- Token support: Support for ERC20 and ERC721 tokens.
+
+- Decoded views: Contract method calls and logs are decoded to provide additional information directly from the UI.
+
+## **Future Development Roadmap**
+
+The following milestones are not included in the collaboration grant assessment and is only here to communicate Subview team's vision and direction.
+
+### Subview **1.1**
+
+- Full-featured API: GraphQL provides the opportunity to test API calls directly from a web interface, and RPC methods are well documented.
+
+### Subview **1.2**
+
+- DEX Info Support: Allow users to view the statistics of dex (similar to uniswap).
+
+## **Colabriation Funding**
+
+Total funding amount: 5,000 USDT.
```

Comment:
We generally prefer to half RING, half USDT, is that ok with you?

Reply:
Of course.